sycl/test-e2e/Matrix/runtime_query_tensorcores.cpp

```diff
@@ -0,0 +1,118 @@
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_70 -o %t.out
```

Comment:
nit, this arch flag isn't necessary for this test, you can use the default which means it will work on all supported devices.
```suggestion
// RUN: %{build} -o %t.out
```

Reply:
done